beetsplug/play.py

```diff
@@ -117,35 +111,76 @@ def play_music(self, lib, opts, args):
                 paths = [relpath(path, relative_to) for path in paths]
             item_type = 'track'
 
-        item_type += 's' if len(selection) > 1 else ''
-
         if not selection:
             ui.print_(ui.colorize('text_warning',
                                   u'No {0} to play.'.format(item_type)))
             return
 
+        open_args = self._playlist_or_paths(paths)
+        command_str = self._create_command_str(opts.args)
+
+        # If user aborts due to long playlist:
+        cancel = self._print_info(selection, command_str, open_args, item_type)
+
+        # Otherwise proceed with play command.
+        if not cancel:
+            play(command_str, paths, open_args)
+
+    def _create_command_str(self, args=None):
+        """Creates a command string from the config command and optional args.
+        """
+        command_str = config['play']['command'].get()
+        if not command_str:
+            return util.open_anything()
+        # Add optional arguments to the player command.
+        if args:
+            if ARGS_MARKER in command_str:
+                return command_str.replace(ARGS_MARKER, args)
+            else:
+                return u"{} {}".format(command_str, args)
+        else:
+            # Don't include the marker in the command.
+            return command_str.replace(" " + ARGS_MARKER, "")
+
+    def _playlist_or_paths(self, paths):
+        """Returns either the raw paths of items or a playlist of the items.
+        """
+        raw = config['play']['raw'].get(bool)
+        if raw:
+            return paths
+        else:
+            return [self._create_tmp_playlist(paths)]
+
+    def _print_info(self, selection, command_str, open_args,
+                    item_type='track'):
+        """Prompts user whether to continue if playlist exceeds threshold.
```

Comment:
I know this is minor, but docstrings should typically be written in the imperative voice—here, "Prompt" rather than "Prompts."